### ISSUE TITLE
Token snap-to-grid does not center token on cursor

### ISSUE THREAD

**Issue by joebobmiles** [repeated-pleasant-games/tabletop-old#55]

**Comment by ToBinio** [comment]
Fix: 
use the mouse-pos instead of the actor-pos to calculate the new snapped char pos
"token.tsx line: 93/96"

probaly you also need to add 8 to the division so
```tsx
 const snap = (value: number) => Math.floor(value / cellSize) * cellSize;
 ```
 to
 ```tsx
const snap = (value: number) => Math.floor((value + 8) / cellSize) * cellSize;
```
"token.tsx line: 25"

**Comment by joebobmiles** [comment]
:postal_horn: Huzzah! :postal_horn:

This issue is resolved in 1.0.0-alpha.12!
